src/main.js

```diff
@@ -20,6 +20,7 @@ import api from "./shared/api.json";
 import oidc from "./shared/oidc.json";
 import version from "./version";
 import { getContextPath } from "./shared/utils";
+import EventBus from './shared/eventbus';
```

Comment:
```suggestion
```

Remove unused import (from an earlier experiment of my while attempting this feature).